UnityProject/Assets/Scripts/UI/UITileListItem.cs

```diff
@@ -36,9 +36,9 @@ public LayerTile Tile
 			item = null;
 			tile = value;
 			itemSpriteRenderer = null;
-			tileSprite = tile.PreviewSprite;
+			tileSprite = tile?.PreviewSprite;
```

Comment:
what does this do?
(the questionmark)

Reply:
Helps avoid NREs that happened when you alt-clicked space tiles

Reply:
Interesting, should read up on how that works out...

Reply:
it's just a short form for `tile != null? tile.PreviewSprite : null`

Reply:
Thanks for the tip! :)